### PR TITLE
Add gettext package for MacOS scripts

### LINKED ISSUE #2780: Mac build broken
Run build on clean Mac

```
/Library/Developer/CommandLineTools/usr/bin/make -C dist pot
gen: fheroes2.pot
make[2]: xgettext: No such file or directory
make[2]: *** [pot] Error 1
make[1]: *** [all] Error 2
make: *** [all] Error 2
```

### PR DESCRIPTION
**Pull request by kant2002**
Closes #2780

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ Windows
 
 MacOS and Linux
 -------------------
+- For Mac run `brew install gettext`
```
Comment (ihhub) [inline README.md:48]: This is actually valid for Ubuntu as well :)

Comment (kant2002) [reply README.md:48]: You can tell me all you like, but I believe that Ubuntu does not have Brew. I do not haves access to Ubuntu, so cannot check what I place proper commands

Comment (ihhub) [reply README.md:48]: I think the right way would be to modify installation scripts for macos to include this package. We need to always think that people don't like to do a lot of steps for setup. Could you please test if it's possible?

Comment (ihhub) [reply README.md:48]: By scripts I meant only for MacOS.

Comment (ihhub) [reply README.md:48]: So we essentially need to modify script/macos/install_sdl1.sh and script/macos/install_sdl2.sh files to add this command. I hope it'll work even with installed gettext.

I wasn't clear in my first post: I meant that some Ubuntu variations do not have gettext package installed as well but this is another issue and here lets focus only on macos.

**Comment by ihhub** [comment]
@kant2002 , thanks a lot for fixing MacOS build!

**Pull request merged**